kata-webhook/create-certs.sh

```diff
@@ -10,7 +10,7 @@ WEBHOOK_SVC="${WEBHOOK_NAME}-webhook"
 
 # Create certs for our webhook
 openssl genrsa -out webhookCA.key 2048
-openssl req -new -key ./webhookCA.key -subj "/CN=${WEBHOOK_SVC}.${WEBHOOK_NS}.svc" -out ./webhookCA.csr 
+openssl req -new -key ./webhookCA.key -subj "/CN=${WEBHOOK_SVC}.${WEBHOOK_NS}.svc" -out ./webhookCA.csr
```

Comment:
Can you get rid of this change? Looks like an erroneous space introduced here.

Reply:
@amshinde the original has an extra space at the end, this change simply removed it.